### PR TITLE
fix: turn off internal topic for transient queries

### PR DESCRIPTION
**Pull request by wcarlson5**
### Description 
turn off internal topic for transient queries so the cleaner can find them
### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Review by cprasad1 — CHANGES_REQUESTED** [review]
@wcarlson5 can we add a test that compares the names of internal topics with the old runtime vs the new runtime? Let's make sure that we don't accidentally change the names of the internal topics

File: ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java [hunk 1]
```diff
@@ -579,7 +579,7 @@ public static Map<String, Object> buildStreamsProperties(
         ? queryId.get().toString()
         : "query";
 
-    if (config.getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED)) {
+    if (config.getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED) && type.equals("query")) {
```
Comment (cprasad1) [inline ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java:582]: can we avoid hardcoding the `query`? possibly via some global variable.config

Comment (ableegoldman) [reply ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java:582]: +1, let's make it a constant in `QueryApplicationId` maybe?

Comment (ableegoldman) [reply ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java:582]: Granted, there are a few other places (that I know of) where we would probably want to switch over to using constants, so we could also just merge this one as-is and make a pass to replace hardcoded strings with a constant in a followup PR

**Review by ableegoldman — APPROVED** [review]
LGTM, I'd say merge this guy when the build completes and then follow up about the constant/hard-coded string stuff -- just so we have one fewer PR floating around and needing to go into the release

File: ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java (hunk 1 above)
Comment (wcarlson5) [reply ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java:582]: Its okay the build didn't kick off so I need to push again anyways

**Comment by wcarlson5** [comment]
@cprasad1 I will add a test for that in a follow up, I think we should get this merged asap

**Review by cprasad1 — APPROVED** [review]

**Pull request merged**